### PR TITLE
bts field now respects passed-in startTime

### PR DESCRIPTION
**Pull request by pbomb**

### CODE REVIEW

File: src/aggregator.js [hunk 1]
```diff
@@ -464,11 +464,20 @@ Aggregator.prototype._getUniqueId = function() {
  * @param {Number} timestamp Timestamp to be converted
  * @private
  */
-Aggregator.prototype._getRelativeTime = function(timestamp) {
+Aggregator.prototype.getRelativeTime = function(timestamp) {
     return (timestamp || new Date().getTime()) - this._startingTime;
 };
 
 /**
+ * Gets the current timestamp relative to the starting time
```
Comment (songawee) [inline src/aggregator.js]: Does this comment need to be updated? getUnrelativeTime: "gets the current timestamp relative..." maybe copy-pasta from the getRelativeTime fn above?

File: src/aggregator.js [hunk 2]
```diff
@@ -464,11 +464,20 @@ Aggregator.prototype._getUniqueId = function() {
  * @param {Number} timestamp Timestamp to be converted
  * @private
```
Comment (songawee) [inline src/aggregator.js]: without the underscore in the _fnName, is this now a public fn?

**Comment by songawee** [comment]
after comments, :shipit:

File: src/aggregator.js (hunk 2 above)
Comment (songawee) [inline src/aggregator.js]: i made it public so it could be tested better

**Comment by pbomb** [comment]
closing because it's out of date now.

**Pull request closed without merging**